src/test/java/it/gov/pagopa/arc/dto/mapper/MapperUtilitiesTest.java

```diff
@@ -81,4 +82,23 @@ void givenWrongDateStringWhenCallDateStringToZonedDateTimeThenReturnException(){
                 () -> MapperUtilities.dateStringToZonedDateTime(wrongDateString));
         Assertions.assertEquals("Invalid date format",exception.getMessage());
     }
+
+    @ParameterizedTest
+    @MethodSource("valueSource")
+    void givenCalculateTotalAmountWhenThen(Long amount, Long fee, Long totalAmount) {
+        //when
+        Long totalAmountResult = MapperUtilities.calculateTotalAmount(amount, fee);
+        //then
+        assertEquals(totalAmount, totalAmountResult);
```

Comment:
let's use more specific names in order to be less ambigous
```suggestion
        assertEquals(expectedTotalAmount, totalAmountResult);
```